build.sbt

```
@@ -21,7 +21,7 @@ val magnoliaVersion = "1.0.0-M4"
 
 val algebirdVersion = "0.13.9"
 val avroVersion = Option(sys.props("avro.version")).getOrElse("1.10.2")
-val bigqueryVersion = "v2-rev20210910-1.32.1"
+val bigqueryVersion = "v2-rev20220123-1.32.1"
```

```suggestion
val bigqueryVersion = "v2-rev20211129-1.32.1"
```

let's keep it in line with [Scio](https://github.com/spotify/scio/blob/main/build.sbt)/Beam for now